packages/mobile/src/components/app-navigator/BottomTabNavigator.tsx

```diff
@@ -75,9 +89,11 @@ const ExploreStackScreen = createStackScreen<ExploreStackParamList>(Stack => (
       name='TrendingUnderground'
       component={TrendingUndergroundScreen}
     />
-    <Stack.Screen name='UnderTheRadar'>
-      {() => <SmartCollectionScreen smartCollection={UNDER_THE_RADAR} />}
-    </Stack.Screen>
+    {smartCollections.map(collection => (
+      <Stack.Screen name={collection.screen} key={collection.screen}>
+        {() => <SmartCollectionScreen smartCollection={collection} />}
```

Comment:
so one thing im thinking we can do, is add `collection` to the `initialParams` to the SmartCollectionScreen, to avoid having to do a child render, so something like

```
<Stack.Screen name={name} key={name} component={SmartCollectionScreen} initialParams={{collection}} />
```
then we can capture those in the smart collection screen like 
```
const {params} = useRoute()
```
i suppose one of the problems here is that we don't know which route (screen name) we are on which is needed as a type arg to useRoute... but food for thought